src/nebari_plugin_self_registration/terraform/modules/self-registration/chart/values.yaml

```
@@ -4,7 +4,7 @@
 
 replicaCount: 1
 image:
-  repository: ghcr.io/metrostar/nebari-self-registration
+  repository: quay.io/nebari-dev/nebari-self-registration
```

The quay.io [organisation](https://quay.io/organization/nebari) is named `nebari`

```suggestion
  repository: quay.io/nebari/nebari-self-registration
```